app/setup.py

```diff
@@ -376,15 +394,17 @@ def setup_publisher(application):
         application.eq["publisher"] = LogPublisher()
 
     else:
-        raise Exception("Unknown EQ_PUBLISHER_BACKEND")
+        raise UnknownEnviormentVariable("Unknown EQ_PUBLISHER_BACKEND")
 
 
 def setup_feedback(application):
     if application.config["EQ_FEEDBACK_BACKEND"] == "gcs":
         bucket_name = application.config.get("EQ_GCS_FEEDBACK_BUCKET_ID")
 
         if not bucket_name:
```

Comment:
Same as above, this could be simplified

Reply:
Updated and  if not is removed 👍

Reply:
Updated : Mebin suggested to keep the if not. But i have added walrus operator to the if not 👍